test/estimation.jl

```diff
@@ -21,25 +21,9 @@ using DSP, Test
 end
 
 @testset "jacobsen" begin
-    # real input: test at two arbitrary frequencies
+    # test at two arbitrary frequencies
     fs = 100
     t = range(0, 5, step = 1/fs)
-    fr = 28.3
-    sr = cos.(2π*fr*t .+ π/4.2)
-    f_est_real = jacobsen(sr, fs)
-    @test isapprox(f_est_real, fr, atol = 1e-5)
-    fr = 12.45
-    sr = sin.(2π*fr*t .+ 3π/2.2)
-    f_est_real = jacobsen(sr, fs)
-    @test isapprox(f_est_real, fr, atol = 1e-5)
-    # test at higher extreme of DFT
-    fr = 49.9002
-    sr = cos.(2π*fr*t)
-    f_est_real = jacobsen(sr, fs)
-    @test isapprox(f_est_real, fr, atol = 1e-5)
```

Comment:
These tests seem useful, assuming they pass. If they don't pass we should probably try to make them. Or is there a fundamental reason they cannot? Then the documentation of `jacobsen` should probably mention this limitation.

Reply:
These tests all pass, and I can add back if you'd prefer. The reason to remove them is that, in general, Jacobsen (like other estimators designed for complex signals) shouldn't be used with real signals, since the results can be quite inaccurate.

Would it be better to enforce that the signal is complex? I didn't do that because I wanted the argument types to be consistent with our ESPRIT implementation.